src/views/Login.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div class="login-background">
+    <v-app>
+      <v-container fluid fill-heignt class="loginOverlay">
+        <v-layout flex justify-start></v-layout>
+        <v-layout flex align-center offset-xs5 fill-height justify-center>
+          <v-flex xs11 sm6 elevation-6>
+            <LoginCard />
+          </v-flex>
+        </v-layout>
+      </v-container>
+    </v-app>
+  </div>
+</template>
+
+<script>
+import LoginCard from "../components/LoginCard.vue";
+export default {
+  name: "Login",
+  components: {
+    LoginCard
+  }
+};
+</script>
+
+<style>
+.login-background {
```

Comment:
We can remove this style I think because the default background is white anyways.